README.md

```diff
@@ -39,6 +38,10 @@ For better support, please include the steps you performed and the result. Also
 
 This add-in is created by Thomas Axelsson.
 
+## Contributors
```

Comment:
Added a contributor section so anyone else who helps can also be listed, its just nice to have contributions, no matter how small, recognised :)